src/routes/routes.test.tsx

```diff
@@ -56,12 +56,12 @@ describe('extractPrefixedSafeAddress', () => {
     expect(extractPrefixedSafeAddress()).toStrictEqual({ shortName, safeAddress: validSafeAddress })
   })
 
-  it('returns the current chain prefix with safe address when a malformed chain is supplied', () => {
+  it('returns the malformed chain prefix with safe address when a malformed chain is supplied', () => {
```

Comment:
This needs to check that it redirects. I think the adjustment you made just makes it a duplicate of the one above it.

Reply:
Right, I will assert that it redirected